uamqp/management_link.py

```diff
@@ -198,7 +198,7 @@ def execute_operation(
         :paramtype type: bytes or str
         :keyword str locales: A list of locales that the sending peer permits for incoming
          informational text in response messages.
-        :keyword float timeout: Provide an optional timeout in milliseconds within which a response
+        :keyword float timeout: Provide an optional timeout in seconds within which a response
```

Comment:
We don't need to convert the value here? Were we already treating it as seconds?